mne/channels/_standard_montage_utils.py

```diff
@@ -334,3 +334,46 @@ def _read_brainvision(fname, head_size):
         pos *= head_size / np.median(np.linalg.norm(pos, axis=1))
 
     return make_dig_montage(ch_pos=_check_dupes_odict(ch_names, pos))
+
+
+def _read_csv(fname, delimiter=','):
```

Comment:
not clear to me why `delimiter` is exposed here.  It's a private function (so users can't actually change it) and it's only called once in the codebase and there's no triaging based on file extension (csv vs tsv).  Do you know there to be XYZ channel location text files that use a different delimiter?  If so, we should expose delimiter in the *public* API somehow.  If not, we should assume comma-delimiter and remove this param.

Reply:
`delimiter` is a leftover parameter, the only different ones are probably tabs in TSV files. Should this be checked in `read_custom_montage` or in `_read_csv`? As for the channel names, I've seen files with channel names (mainly CSV files) and ones without (mainly XYZ files).